dll/cpl/console/lang/fr-FR.rc

```diff
@@ -46,7 +46,7 @@ BEGIN
     LTEXT "&Police :", IDC_STATIC, 10, 105, 33, 10
     AUTOCHECKBOX "&Gras", IDC_CHECK_BOLD_FONTS, 56, 105, 60, 10
     LISTBOX IDC_LBOX_FONTTYPE, 10, 120, 110, 40, LBS_SORT | LBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX "Selected Font", IDC_GROUPBOX_FONT_NAME, 7, 156, 240, 50
+    GROUPBOX "Police sélectionnée", IDC_GROUPBOX_FONT_NAME, 7, 156, 240, 50
```

Comment:
Please check that all these labels display correctly.

Reply:
Yes. With proposed modifications I have to adapt a bit but i'll post "post modification" screenshots.